component/src/main/ballerina/ftp/client_action.bal

```diff
@@ -0,0 +1,114 @@
+
+    documentation {
+        The `delete()` function can be used to delete a file from an FTP server.
+
+        P{{path}} The resource path
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function delete(string path) returns error?;
+
+    documentation {
+        The `put()` function can be used to add a file to an FTP server.
+
+        P{{path}} The resource path
+        P{{channel}} A ByteChannel that represent the local data source
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function put(string path, io:ByteChannel channel) returns error?;
+
+    documentation {
+        The `append()` function can be used to append content to an existing file in an FTP server.
+        A new file will create if file does not exist.
+
+        P{{path}} The resource path
+        P{{channel}} A ByteChannel that represent the local data source
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function append(string path, io:ByteChannel channel) returns error?;
+
+    documentation {
+        The `mkdir()` function can be used to create new direcotry in an FTP server.
+
+        P{{path}} The directory path
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function mkdir(string path) returns error?;
+
+    documentation {
+        The `rmdir()` function can be used to delete an empty direcotry in an FTP server.
+
+        P{{path}} The directory path
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function rmdir(string path) returns error?;
+
+    documentation {
+        The `rename()` function can be used to rename a file or move to new location with in the same FTP server.
+
+        P{{origin}} The source file location
+        P{{destination}} The destination file location.
+        R{{}} Returns an `error` if failed to establish communication with the FTP server
+    }
+    public native function rename(string origin, string destination) returns error?;
+
+    documentation {
+        The `size()` function can be used to get the size of a file resource.
+
+        P{{path}} The resource path
+        R{{}} The file size in bytes or an `error` if failed to establish communication with the FTP server
+    }
+    public native function size(string path) returns int|error;
+
+    documentation {
+        The `list()` function can be used to get the file name list in a give folder.
+
+        P{{path}} The direcotry path
+        R{{}} An array of file names or an `error` if failed to establish communication with the FTP server
+    }
+    public native function list(string path) returns string[]|error;
+
+    documentation {
+        The `isDirectory()` function can be used checks if given resource is a direcotry.
```

Comment:
Change to 'The `isDirectory()` function can be used to check if a given resource is a directory.'